src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import React, { FC } from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (a: number) => void;
+}
+
+export const Pagination: FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  Pagination.defaultProps = {
+    currentPage: 1,
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            onPageChange(currentPage - 1);
+          }}
```

Comment:
suggest to make one fuction factory and use it in all places
```
const handlePageChange = (page) => () => onPageChange(page);
```
```suggestion
          onClick={handlePageChange(currentPage - 1)}
```